tests/Tests/SKShaperTest.cs

```diff
@@ -58,7 +58,7 @@ public void CorrectlyShapesArabicScript()
 		{
 			var clusters = new uint[] { 4, 2, 0 };
 			var codepoints = new uint[] { 629, 668, 891 };
-			var points = new SKPoint[] { new SKPoint(0, 0), new SKPoint(28.375f, 0), new SKPoint(42.125f, 0) };
+			var points = new SKPoint[] { new SKPoint(0, 0), new SKPoint(28.25f, 0), new SKPoint(42, 0) };
```

Comment:
This is interesting. It appears that the characters have shifted a tiny bit to the left by 0.125.